include/integratorxx/c_api.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
```

Comment:
It might be a good idea to use the standard C approach here, as `#pragma once` appears to be compiler dependent..